res/xml/device_maintainers_fragment.xml

```diff
@@ -1886,6 +1886,13 @@
             app:codename="@string/device_sunfish_codename"
             app:maintainer="@string/device_sunfish_maintainer"
             app:type="phone" />
+	    
+	<com.android.settings.rr.Preferences.DevicePreference
```

Comment:
wrong category. it shouldn't be under google devices

Reply:
Thank you very much.